### DISCUSSION TITLE
Nitpick: Apple Watch key color?

### DISCUSSION THREAD

**Discussion by cerahmed** [kormax/apple-home-key-reader#31] [Q&A] [answered]
Hey Kormax,

Still expirementing with this amazing lib. Figured out to point out one minor issue that I encountered.
I've set the key color as black. It shows correct on the iPhone, however it remains the default (platinum?) color on my Apple Watch.

Not sure if it's the codebase or Apple's way of displaying the key on the Watch; googled a few watch key images and seems like it could indeed be black. Any ideas?

P.S. again, it's a nitpick, not really bothering.

**Comment by kormax** [comment, marked as answer]
Hello.

The HARDWARE_FINISH is a static HAP property that devices use in order to decide what key color to display.

There's no additional logic related to that inside of this codebase, everything else happens in the OS.

I've also encountered an issue with key color on a watch being desynchronized from a phone, with the issue resolving itself and reappearing sometimes. To me it seems like an OS bug, as there are no other clues, and phones work perfectly.

**Reply by cerahmed** [reply]
Agreed. Seems the functionality works as expected on iOS as you mentioned. Just wanted to check if I'm the only one. Thanks for the prompt response.
